src/focus-ring.js

```diff
@@ -74,11 +80,13 @@ document.addEventListener('DOMContentLoaded', function() {
    * opening a menu or dialog.
    */
   function onKeyDown() {
-    hadKeyboardEvent = true;
     // `activeElement` defaults to document.body if nothing focused,
     // so check the active element is actually focused.
-    if (matches(document.activeElement, ':focus'))
-      addFocusRingClass(document.activeElement);
+    var activeElement = document.activeElement;
+    if (activeElement.tagName == 'BODY')
+      return;
+
+    hadKeyboardEvent = true;
```

Comment:
Ah - I had it backwards. However, since the keyboard event precedes the focus event, this is checking where focus is coming _from_, rather than going _to_, isn't it?